launch/dynamo-run/src/flags.rs

```diff
@@ -92,6 +95,13 @@ pub struct Flags {
     #[arg(long)]
     pub leader_addr: Option<String>,
 
+    /// If using `out=dyn://..` with multiple backends, this says how to route the requests.
+    ///
+    /// Mostly interesting for KV-aware routing.
+    /// Defaults to RouterMode::Random
+    #[arg(long, default_value = "random")]
+    pub router_mode: RouterMode,
```

Comment:
Just curious - is there a world where it makes sense to use the standalone `router` binary recently added (https://github.com/ai-dynamo/dynamo/blob/main/components/router/src/main.rs) (though I think it's currently only focused on KV routing and doesn't expose random/round-robin) as a sidecar piece rather than embedded it in dynamo-run? 

ex:
```
dynamo run ... out=dyn://...
...

./router_binary ...
```

Just trying to reconcile the many ways we seem to have of doing many things currently.

Reply:
@ryanolson ^